src/lib/select/select.ts

```diff
@@ -124,7 +127,19 @@ export class MdSelect implements AfterContentInit, ControlValueAccessor, OnDestr
   private _placeholder: string;
 
   /** The animation state of the placeholder. */
-  _placeholderState = '';
+  private _placeholderState = '';
+
+  get placeholderState(): string {
```

Comment:
Can you return this to an internal method or property with `_`? This shouldn't be in the public API.